tests/php/TestCase.php

```diff
@@ -0,0 +1,99 @@
+<?php # -*- coding: utf-8 -*-
+
+namespace Mollie\WooCommerce\Tests;
+
+use function Brain\Monkey\setUp;
+use function Brain\Monkey\tearDown;
+use Mockery;
+use PHPUnit\Framework\TestCase as PhpUniTestCase;
```

Comment:
What do you mean exactly? Where do you see PSR-0?

Reply:
Right below, on line 9.

Reply:
`PHPUnit_Framework_MockObject_MockBuilder` is part of `phpunit-mock-object` and doesn't have any NameSpace.

Reply:
Yea, I know it doesn't have a namespace, because it's PSR-0. And PHPUnit is PSR-4. Did the newer PHPUnit with PSR-4 really depend on an older mock object lib with PSR-0? Or is there some kind of problem there?

Reply:
I think it's depend on the version used of phpunit if you go to https://github.com/sebastianbergmann/phpunit-mock-objects you'll see the repository was archived by the owner, if you read the readme

> phpunit-mock-objects is the default mock object library for PHPUnit.

Problem is we have to support php 5.6 and the phpunit installed by composer is phpunit@^5, probably the phpunit-mock-objects was removed as a dependency later.

Phpunit is PSR-4 but phpunit-mock-objects is a dependency not part of the phpunit package